internal/portforward/service/command.go

```diff
@@ -0,0 +1,32 @@
+package service
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+func (s *Service) runUpCommand(ctx context.Context, ports []uint16) (err error) {
```

Comment:
I'll add a test to make sure for example this works: `/bin/sh -c "echo {{PORTS}}"` - given it plugs arguments by splitting them by spaces, I'm not sure it would handle that case with double quotes correctly. Will check!